oasislmf/execution/bin.py

```diff
@@ -392,6 +399,44 @@ def prepare_run_inputs(analysis_settings, run_dir, ri=False):
         raise OasisException("Error preparing the model 'inputs' directory: {}".format(e))
 
 
+@oasis_log
+def set_footprint_set(setting_val, run_dir):
+    """
+    Create symbolic link to footprint file set that will be used for output
+    calculation.
+
+    :param setting_val: identifier for footprint set
+    :type setting_val: string
+
+    :param run_dir: model run directory
+    :type run_dir: string
```

Comment:
Minor: args and returns don't follow usual format with Args:... and Returns:...

Reply:
I followed the style that was already used in other places in the file. I notice that there are a few other files that follow the same style. Perhaps we should create a new issue to standardise all these files?

Reply:
Yes, there is confusion all over the place. Fine if you want to keep them like they are.
Anyway, some time ago we introduced [CONTRIBUTING.md](https://github.com/OasisLMF/OasisLMF/blob/main/CONTRIBUTING.md) guidelines where we decided to follow Google-style docstrings, see the referred [Chapter 3.8](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings).

Please go ahead and open an issue to standardise all the docstrings, I'm in favour of that.

Reply:
Please see issue https://github.com/OasisLMF/OasisLMF/issues/1356.